client/containers/Pub/PubDocument/FileImport/useFileManager.js

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+
+import { s3Upload } from '../../../../utils';
```

Comment:
I believe this can just be `from 'utils'` as it is aliased in our webpack config.